notes/2019/2019-02-28.md

```diff
@@ -0,0 +1,115 @@
+
+Additionally, the following individuals are not in attendance but support consensus:
+
+* Gyandeep Singh (@gyandeeps) - TSC
+* Kai Cataldo (@kaicataldo) - TSC
+
+## Topics
+
+### [Fix: no-var fixed to incorrect code (fixes #11441)](https://github.com/eslint/eslint/pull/11443)
+
+* **Summary**: When using the TypeScript parser, the fixer for `one-var` can replace `declare var foo: number;` with `let var foo: number;` because it assumes the first token is the `var` keyword without actually verifying that.
+* **Question**: Though everything works okay using the default parser, should we still accept the change as a bugfix to improve compatibility with custom parsers?
+* :+1: from @nzakas, @btmills, @not-an-aardvark, @platinumazure
+
+### [v6.0.0](https://github.com/eslint/eslint/projects/5)
+
+#### Drop Node.js 6 support
+
+* Release does not necessarily need to come after Node.js 6 EOL in April, though it likely will.
+* :+1: from @nzakas, @btmills, @not-an-aardvark, @platinumazure
+
+**Resolution**: We will drop support for Node.js 6. @not-an-aardvark will create an issue to discuss minimum versions of the supported major releases.
+
+#### [eslint-recommended changes in eslint v6.0.0](https://github.com/eslint/eslint/issues/10768)
+
+* @nzakas proposes marking the issue as accepted and discussing the exact rule changes on the issue.
+* :+1: from @nzakas, @btmills, @not-an-aardvark, @platinumazure
+
+**Resolution**: We will accept breaking changes to `eslint:recommended` with specifics to be determined.
+
+#### [Throw an error for invalid `globals` values](https://github.com/eslint/eslint/pull/11338#discussion_r252497158)
+
+* :+1: from @btmills, @platinumazure, @nzakas, @not-an-aardvark
+
+**Resolution**: This is accepted as a breaking change in v6.0.0.
+
+#### [do not ignore files started with `.` by default](https://github.com/eslint/eslint/issues/10341)
+
+* Linting directories like `.git` by default would be confusing.
+* Users can still manually enable linting dot-prefixed files and directories.
+* Coming up with a list of one-size-fits-all default exceptions would be difficult.
+* @mysticatea and @kaicataldo left :+1: on the issue
+* :-1: from @platinumazure
+* @nzakas suggests tabling for v6 and discussing later. :+1: from @platinumazure, @btmills, @not-an-aardvark
+
+**Resolution**: This proposal is tabled for discussion after 6.0.0.
+
+#### [Stricter `no-undef` default](https://github.com/eslint/eslint/issues/10203)
+
+* Currently, `typeof foo` checks when `foo` is not defined will not be caught unless the user has enabled the `typeof` option.
+* This proposes enabling the `typeof` option by default so that `typeof foo` would cause a warning unless the user disables the option.
+* @platinumazure is against :+1: because `typeof foo` is permitted by the language spec. `typeof` checks do not throw a `ReferenceError` at runtime. Further, since users can already opt into the check, this would be a breaking change without significant value.
```

Comment:
Should this say `:-1:`?

```suggestion
* @platinumazure is against :-1: because `typeof foo` is permitted by the language spec. `typeof` checks do not throw a `ReferenceError` at runtime. Further, since users can already opt into the check, this would be a breaking change without significant value.
```